pages/tickets.js

```diff
@@ -0,0 +1,35 @@
+import Ticket from "../components/Ticket/index";
+import FormTicket from "../components/TicketForm/index";
+
+import styled from "styled-components";
+
+export default function CreateTicket({ onAddTicket, tickets, onDeleteTicket }) {
+  return (
+    <>
+      <WrapperDiv>
+        <FormTicket onAddTicket={onAddTicket} />
+        <TicketList>
+          {tickets?.map((ticket, index) => (
+            <Ticket
+              key={index}
```

Comment:
Same applies here, you need a stable id.

Reply:
Copy